lstchain/tools/lstchain_create_drs4_pedestal_file.py

```diff
@@ -0,0 +1,113 @@
+"""
+Create drs4 time pedestal fits file.
+"""
+import numpy as np
+from astropy.io import fits
+from tqdm.autonotebook import tqdm
+
+from ctapipe.core import Provenance, traits
+from ctapipe.core import Tool
+from ctapipe.io import EventSource
+from lstchain.calib.camera.drs4 import DragonPedestal
+
+
+class PedestalFITSWriter(Tool):
+
+    name = "PedestalFITSWriter"
+    description = "Generate a pedestal FITS file"
+
+    output = traits.Path(
+        default_value="pedestal.fits",
+        help="Path to the generated fits pedestal file",
+        directory_ok=False,
+    ).tag(config=True)
+
+    no_deltaT = traits.Bool(
```

Comment:
That's not what I meant :D Having the `deltaT` boolean with default True is perfect! But if a bool is by default True, than you need a flag to switch it off.  So having `deltaT = traits.Bool(default_value=True).tag(config=True)` is correct, but then the flag must be:

```
flags = {'no-delta-t': ({"PedestalFITSWriter": {"deltaT": False}}, "Switch off delta T corrections")}
```

Reply:
The flag must always set the opposite of the default to make sense.